src/containers/ConceptPage/conceptTransformers.ts

```diff
@@ -89,7 +86,7 @@ export const getNewConceptType = (
     processed: values.processed ?? false,
   },
   tags: values.tags,
-  metaImage: metaImageFromForm(values),
+  metaImage: undefined,
```

Comment:
Hvorfor ikke bare fjerne dette helt?

Reply:
Tenkte eg skulle vente med det til backend er ferdig og typene oppdatert. Tas dette inn no trenger vi ikkje vente på det.

Reply:
Kunne visst slette denne ja.

Reply:
Tenker det er greit å vente på at backend-PR'en er inne, så vi får fjernet disse fullstendig. Dette er sånn man fort glemmer å fjerne i etterkant.

Reply:
Eg liker å gjøre det i to steg slik at vi ikkje alltid er avhengig av deploy samtidig av backend og frontend.